packages/flutter/lib/src/foundation/change_notifier.dart

```diff
@@ -62,11 +62,11 @@ abstract class Listenable {
   /// Return a [Listenable] that triggers when any of the given [Listenable]s
   /// themselves trigger.
   ///
-  /// The list must not be changed after this method has been called. Doing so
-  /// will lead to memory leaks or exceptions.
+  /// [listenables] must not be changed after this method has been called.
```

Comment:
Actually, this documentation could be misunderstood that you are not allowed to change the individual `Listenables`. We should make it clear that you are not allowed to change what the Iterable iterates over. Every iteration must yield the same result.

Reply:
Yeah, that's a good point. Hopefully the current documentation is better:

```dart
/// Once the factory is called, items must not be added or removed from the iterable.
/// Doing so will lead to memory leaks or exceptions.
```